analyses/tp53_nf1_score/00-tp53-nf1-alterations.R

```diff
@@ -93,10 +93,7 @@ tp53_coding <- coding_consensus_snv %>%
 # subset to TP53 cnv loss and format to tp53_coding file format
 tp53_loss<-cnvConsesus %>% 
```

Comment:
```suggestion
tp53_loss<-cnvConsensus %>% 
```

noticing some old typos - can you update while you are here?

Reply:
@runjin326 can you update the typos in this script please

Reply:
Sure!

Reply:
@jharenza - ready for another look now!